### ISSUE TITLE
Bring forum server to standard config for access

### ISSUE THREAD

**Issue by DanVanAtta** [triplea-game/triplea#3947]
Checking forum server, the standard root password does not work and I do not have an account, I cannot access it. For this task please:
- [ ] change root password to the standard password
- [ ] run infrastructure cron job setup command so we can have standard config and SSH keys: https://github.com/triplea-game/infrastructure#prepare-the-install-script-in-a-text-editor

**Comment by RoiEXLab** [comment]
@DanVanAtta I don't think the forum server even has a root password, I think I disabled root login via ssh at some time in the past.
I'd be happy to create a user for you so you can have access at the files and have a look at the file system (Although I could have sworn you already have a user on that server).
Same ssh key as on infrastructure?

**Comment by RoiEXLab** [comment]
@DanVanAtta I just had a quick look:
You do have a user on this server configured with an RSA key.
Should I add your current key?

**Comment by prastle** [comment]
Also to clarify here since you closed where I was talking about it @DanVanAtta   Any root can be changed at the linode level  quite easily on a server reboot once securely logged into linode.  So I guess your saying the 5 of us are a security issue?

**Comment by DanVanAtta** [comment]
@RoiEXLab root login with password is not quite the same as the root account not having a password. Once logged into a machine, one can `su` and use password. Having an oddball server not have keys managed the same way, or to get updates, adds to our work (otherwise it would be just one of N machines, with an application that is still hand-managed; thinking back to TripleA's system infrastructure history, where updated jar files were dropped in from folders created years ago, with no information beyond that, I'm eager to see us automate all of this so we can invest minimal time to sys-admin and focus on other priorities)

@prastle interesting, news to me. Well the account is protected, and getting linode account hacked would already be a breach to a pretty trusted level of access. In some ways yes, the 5 of us are a security issue as that is 5 people whose computers you could hijack and then escalate privileges from there after logging in to linode. So I trust your passsword is not "12346" :grin:

**Comment by prastle** [comment]
LOL well if we are worried about it that bad I am truly WORRIED!  Either way I don't save my passwords for stuff like that as "remember me".  Hopefully the rest of you are similar ;) @DanVanAtta

**Comment by DanVanAtta** [comment]
@RoiEXLab @prastle looking to follow-up here. Any objection if we update the root password to be the 'standard' one? Or let's please add a row with the pass to the secrets doc. 

Second, any objection if we run an app-agnostic infrastructure job on the machine @RoiEXLab ? This would just be adding the standard infrastructure cron-job, this would add our SSH keys and the common set of apt-packages and security settings.

**Comment by RoiEXLab** [comment]
Do the infrastructure scripts update the systems automatically? 
If not I don't have any objections on connecting the forum server.
However need to make sure that the scripts treat the forum fundamentally different, so opening the common ports (and closing all other ones for example) would definitely break things.

**Comment by prastle** [comment]
@DanVanAtta  I would sug we just make them all the same password as the "New" variation of the old one that I added with linode to pre release for @ssoloff.

**Comment by prastle** [comment]
@RoiEXLab  yes they appear to in the server thread

**Comment by DanVanAtta** [comment]
> Do the infrastructure scripts update the systems automatically?

Yes, install the cron one-liner that is the same on every machine (except prerelease which uses a different branch), and the common system settings are applied automatically, idempotent, every time the cron runs.

> However need to make sure that the scripts treat the forum fundamentally different, so opening the common ports (and closing all other ones for example) would definitely break things.

UFW is turned on, port 22 is enabled. AFAIK we can add some parameters or config to open up any needed ports. No ports are explicitly closed, but UFW does work on a white list so anything not opened would be closed.


@prastle 
> @DanVanAtta I would sug we just make them all the same root password as the "New" variation of the old one that I added with linode to pre release for @ssoloff.

Yes, not much point in having those be different. Our standard password I'm not sure you have access to the doc or not. Let's chat on gitter about this and I can pass you the URL.

**Comment by prastle** [comment]
@DanVanAtta  I replied to you on your gitter thread.
